### PR TITLE
staking fixes - not quite working yet

### PR DESCRIPTION
**Pull request by tobowers**
this changes around the UI a bit on the earn side to do some of the changes we talked about (getting rid of management within hippodrome).

There's a bug with the adding of liquidity right now that I still need to investigate, but these are the UI changes that we want I believe so figured I'd open a PR.
Screen Shot 2021-07-16 at 00 26 20

### CODE REVIEW

File: packages/hippodrome-ui/src/components/stake/StakeOutputTokensAmount.tsx [hunk 1]
```diff
@@ -5,25 +5,31 @@ import { useTokenQuote } from "../../hooks/useTokenQuote"
...
+    half.toHexString()
   )
   const { amountOut: renDogeAmount, loading: renDogeLoading } = useTokenQuote(
     input,
     RENDOGE_ADDRESS,
-    amount
+    half.toHexString()
   )
 
   console.log(renDogeAmount, "ells", humanBigNumber(renDogeAmount || 0, 18))
```
Comment (alabobriggs) [inline packages/hippodrome-ui/src/components/stake/StakeOutputTokensAmount.tsx:35]: Not your change, I might have added this earlier :) Please help delete

File: packages/hippodrome-ui/src/components/transaction/AwaitingMint.tsx [hunk 2]
```diff
@@ -233,17 +234,16 @@ const DepositStakeConfirmed: React.FC<DepositConfirmedProps> = ({
     try {
       setLoading(true)
       console.log("swapping: ", deposit)
-      await doAddLiquidity(deposit, deposit.lockAndMint.params)
+      await doAddLiquidity(chain, deposit, deposit.lockAndMint.params)
       history.push("/")
       toast({
-        title: "Stake completed",
-        description: "Your stake will appear in next screen",
-        duration: 9000,
```
Comment (alabobriggs) [inline packages/hippodrome-ui/src/components/transaction/AwaitingMint.tsx:241]: this will default to 5000 if it isn't specified

**Review by alabobriggs — APPROVED** [review]
looks good

File: packages/hippodrome-ui/src/components/transaction/AwaitingMint.tsx (hunk 2 above)
Comment (tobowers) [reply packages/hippodrome-ui/src/components/transaction/AwaitingMint.tsx:241]: ah ok - well i'll add back in a *longer* duration then in another PR

**Pull request merged**